### PR TITLE
Feat/collections image

### PR DESCRIPTION
**Pull request by juliswer**

### CODE REVIEW

File: src/collections/repositories/collection.repository.ts [hunk 1]
```diff
@@ -12,30 +13,106 @@ export class CollectionRepository extends Repository<Collection> {
...
+    return {
+      totalCount,
+      collections,
+    };
+  }
+
+  async pagination(
+    paginationDto: CollectionPaginationDto
+  ): Promise<CollectionResponseDto | undefined> {
+    const { limit, skip, creationOrder, mintType, popularityOrder } = paginationDto;
```
Comment (GimerStackIt) [inline src/collections/repositories/collection.repository.ts:42]: Check this warning: minType is never used.

File: src/items/repositories/item.repository.ts [hunk 2]
```diff
@@ -326,4 +404,38 @@ export class ItemRepository extends Repository<Item> {
...
+      condition1.name = Raw(
+        (alias) => `LOWER(${alias}) Like '%${paginationDto.search?.toLowerCase()}%'`
+      );
+
+      condition2.description = Raw(
+        (alias) => `LOWER(${alias}) Like '%${paginationDto.search?.toLowerCase()}%'`
+      );
+    }
+
+    // This query will result on the following structure:
```
Comment (GimerStackIt) [inline src/items/repositories/item.repository.ts]: Get rid of these comments

**Review by GimerStackIt — COMMENTED** [review]
I added a few comments

File: src/collections/repositories/collection.repository.ts (hunk 1 above)
Comment (juliswer) [reply src/collections/repositories/collection.repository.ts:42]: Yep, its ready for when the lazy minting feature is done

File: src/items/repositories/item.repository.ts (hunk 2 above)
Comment (juliswer) [reply src/items/repositories/item.repository.ts]: These comments works to explain better what was done above

**Review by mmiranda-stackit — DISMISSED** [review]

**Review by mmiranda-stackit — DISMISSED** [review]

**Review by mmiranda-stackit — APPROVED** [review]

**Pull request merged**